profiles/pci/graphic_drivers/profiles.toml

```diff
@@ -132,7 +132,7 @@ class_ids = "0300 0302"
 vendor_ids = "8086"
 device_ids = "*"
 priority = 4
-packages = 'mesa lib32-mesa libva-intel-driver lib32-libva-intel-driver opencl-clover-mesa lib32-opencl-clover-mesa vulkan-intel lib32-vulkan-intel intel-media-driver mesa-vdpau lib32-mesa-vdpau'
+packages = 'mesa lib32-mesa libva-intel-driver lib32-libva-intel-driver vulkan-intel lib32-vulkan-intel intel-media-driver intel-compute-runtime intel-oneapi-compiler-shared-runtime mesa-vdpau lib32-mesa-vdpau'
```

Comment:
Is it needed outside of the AI profile?

Reply:
Yeah installing just these packages made intel opencl target appear in indigobench